From 722ea58e1523a4dafec880591e9a8a8a80510af6 Mon Sep 17 00:00:00 2001
From: Doug Ledford <dledford@redhat.com>
Date: Wed, 24 Sep 2014 15:22:23 -0400
Subject: [redhat] configs: Enable ocrdma driver

Message-id: <d065150acf26dbd513b57dadba96df2ab2f059c2.1411571481.git.dledford@redhat.com>
Patchwork-id: 96641
O-Subject: [Patch RHEL7 48/48] Enable ocrdma driver
Bugzilla: 1081067 1088577
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

But don't build on s390x

Signed-off-by: Doug Ledford <dledford@redhat.com>
---
 create mode 100644 redhat/configs/generic/s390x/CONFIG_INFINIBAND_OCRDMA

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/CONFIG_INFINIBAND_OCRDMA b/redhat/configs/generic/CONFIG_INFINIBAND_OCRDMA
index 12ff351..5f6ad2a 100644
--- a/redhat/configs/generic/CONFIG_INFINIBAND_OCRDMA
+++ b/redhat/configs/generic/CONFIG_INFINIBAND_OCRDMA
@@ -1 +1 @@
-# CONFIG_INFINIBAND_OCRDMA is not set
+CONFIG_INFINIBAND_OCRDMA=m
diff --git a/redhat/configs/generic/s390x/CONFIG_INFINIBAND_OCRDMA b/redhat/configs/generic/s390x/CONFIG_INFINIBAND_OCRDMA
new file mode 100644
index 0000000..12ff351
--- /dev/null
+++ b/redhat/configs/generic/s390x/CONFIG_INFINIBAND_OCRDMA
@@ -0,0 +1 @@
+# CONFIG_INFINIBAND_OCRDMA is not set
-- 
1.7.1